weaver/common/protos-java-kt/build.gradle

```diff
@@ -15,29 +18,37 @@ buildscript {
 plugins {
     id "maven-publish"
     id 'java'
-    id 'com.google.protobuf' version '0.8.12'
+    id 'com.google.protobuf' version '0.9.4'
     id 'idea'
 }
 
 repositories {
 	mavenLocal()
 	mavenCentral()
 	maven { url 'https://repo1.maven.org/maven2' }
-	maven { url 'https://download.corda.net/maven/corda-dependencies/' }
 	maven { url 'https://jitpack.io' }
 }
 
 apply plugin: 'kotlin'
 
 tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile) {
-    kotlinOptions {
-        languageVersion = "1.3"
-        apiVersion = "1.3"
-        jvmTarget = "1.8"
+    compilerOptions {
+        languageVersion = KOTLIN_1_9
+        apiVersion = KOTLIN_1_9
+        jvmTarget = JVM_17
```

Comment:
Where are we getting these values from?

Reply:
These are imported at the top of the file.